JenkinsfileImageTest

```diff
@@ -0,0 +1,111 @@
+properties(
+        [
+                buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '1000', daysToKeepStr: '', numToKeepStr: '1000')),
+                pipelineTriggers(
+                        [[
+                                 $class: 'CIBuildTrigger',
+                                 checks: [
+                                         [expectedValue: 'Fedora-Cloud$|Fedora$', field: 'release_short'],
+                                         [expectedValue: 'ga', field: 'release_type'],
+                                         [expectedValue: 'FINISHED', field: 'status'],
+                                 ],
+                                 providerName: 'fedora-fedmsg', selector: 'topic = "org.fedoraproject.prod.pungi.compose.status.change"'
+                         ]]
+                ),
+                parameters(
+                        [
+                                string(defaultvalue: "", description: 'CI_MESSAGE', name: 'CI_MESSAGE'),
+                                string(defaultvalue: "", description: 'MESSAGE_HEADERS', name: 'MESSAGE_HEADERS')
+                        ]
+                )
+        ]
+)
+
+
+openshiftProject = "continuous-infra"
+DOCKER_REPO_URL = '172.30.254.79:5000'
+
+pipeline{
+    agent {
+        kubernetes {
+            cloud 'openshift'
+            label 'fedora-image-test'
+            containerTemplate {
+                name 'jnlp'
+                args '${computer.jnlpmac} ${computer.name}'
+                image DOCKER_REPO_URL + '/' + openshiftProject + '/jenkins-continuous-infra-slave:stable'
+                ttyEnabled false
+                command ''
+            }
+        }
+    }
+    stages{
+        stage("set build name"){
+            steps{
+                step([$class: 'WsCleanup'])
+                script{
+                    sh("echo \$CI_MESSAGE > ci_message.txt")
+                    env.compose = sh(script: "cat ci_message.txt | jq -r .compose_id", returnStdout: true).trim()
+                    env.release_version = sh(script: "cat ci_message.txt | jq -r .release_version", returnStdout: true).trim()
```

Comment:
We're going to need one of these per major release and adjust accordingly, or find a way to have the last success have the images for all major releases.

So maybe something like, get all artifacts from lastSuccessful build *except* the release for which this message is for